### ISSUE TITLE
SVG icons doesn't show images inside

### ISSUE THREAD

**Issue by spheras** [godotengine/godot#83198]
### Godot version

4.2-dev6

### System information

Godot v4.2.beta1 - Windows 10.0.22621 - Vulkan (Forward+) - integrated AMD Radeon(TM) Graphics (Advanced Micro Devices, Inc.; 31.0.12044.3) - AMD Ryzen 3 5300U with Radeon Graphics (8 Threads) 

### Issue description

Checking changes on the Godot v4.2.dev6, also just downloaded beta1 and the same, I advertised that the icon I used in my addon was not rendered correctly.  The problem was that this SVG contained a raster image inside and some vectors over.  The raster image (body) disappeared from the rendered SVG.

![image](https://github.com/godotengine/godot/assets/3862933/a50fdc5d-a90f-4935-be48-f8263dcb0d16)

The SVG was created with Inkscape, and it was correctly rendered with the last stable release of Godot 4.1.
This is how it is rendered correctly with 4.1.
![image](https://github.com/godotengine/godot/assets/3862933/520f17ce-f030-4ebc-ace7-bcce176cc138)

### Steps to reproduce

I attached a minimal project, but, really, you only have to get the SVG from there.
Just put the .svg as the icon of the project, and you will see the bad render even in the project manager window.

### Minimal reproduction project

a minimal project to reproduce

[svgtest.zip](https://github.com/godotengine/godot/files/12884066/svgtest.zip)

**Comment by Calinou** [comment]
Please check in https://www.thorvg.org/viewer to confirm whether this is a ThorVG regression.

**Comment by spheras** [comment]
It seems to be rendered perfectly there: 
![image](https://github.com/godotengine/godot/assets/3862933/8a8161aa-c35b-4e79-b126-73f19baa3463)

the picture section is detected and rendered.

**Comment by fire** [comment]
This was one of the features I requested for godot 4.0 will poke around the code and from thorvg. I suspect a jpeg code oath was modified. @hermet

**Comment by hermet** [comment]
the robot-body is desrcibed with png image:

```
    <image
       width="213.51875"
       height="192.88126"
       preserveAspectRatio="none"
       xlink:href="data:image/png;base64,iVBORw0KGgoAAAA ...
```

Probably, the godot disabled the png loader of the thorvg in building.

**Comment by capnm** [comment]
Yea the png loader was disabled, I'll fix it in 0.11.1

**Comment by capnm** [comment]
> ... I suspect a png or jpeg code path was modified.

No the png/jpg loader was disabled because of size complaints, I will re-enable it in 0.11.1
